libs/datasets/combined_datasets.py

```diff
@@ -298,14 +300,14 @@ class Override(Enum):
     BY_TIMESERIES = 2
     # For each <fips, variable, date>, use the highest priority datasource that has a real
     # (not NaN) value.
-    BY_TIMESERIES_POINT = 3
+    BY_TIMESERIES_POINT = 3  # Deprecated
 
 
 def _build_dataframe(
```

Comment:
Maybe `_build_dataframe_with_provenance` be more applicable now

Reply:
i went with slightly shorter `_build_data_and_provenance`